src/App.tsx

```diff
@@ -16,47 +16,93 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  const [
+    sortedGoods,
+    setSortedGoods,
+  ] = useState(goodsFromServer);
+  const [isVisibleReser, setIsVisibleReser] = useState(false);
```

Comment:
I don't quite understand what you meant by that name
```suggestion
  const [isResetButtonActive, setIsResetButtonActive] = useState(false);
```